src/librustdoc/html/render.rs

```diff
@@ -1157,7 +1157,7 @@ themePicker.onblur = handleThemeButtonsBlur;
             &format!("{}\n{}", variables.join(""), all_indexes.join("\n")),
             options.enable_minification),
             &dst);
-        try_err!(write!(&mut v, "initSearch(searchIndex);addSearchOptions(searchIndex);"), &dst);
+        try_err!(write!(&mut v, "addSearchOptions(searchIndex);initSearch(searchIndex);"), &dst);
```

Comment:
Oh, I forgot to note during initial review -- can you explain why this has changed?

Reply:
A global variable story. Not sure if it'll be needed anymore once I apply the new changes you suggested.